### PR TITLE
Updated dev_tools to work with Linux

### PR DESCRIPTION
**Pull request by rAlexandre00**
## Description
Updated the /scripts/dev_tools.sh to use xdg-open instead of open when running on Linux

## Affected Dependencies
None

## How has this been tested?
Ran the script on Ubuntu 20.04 and checked the behavior of the script for http, redis and postgres ports

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Review by iamtrask — APPROVED** [review]

**Comment by rasswanth-s** [comment]
@rAlexandre00 Great work on this PR, would it possible to add also windows support, that would be really helpful.

**Comment by teo-milea** [comment]
great work @rAlexandre00 and congrats on your first contribution!!

File: scripts/dev_tools.sh [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
     PORT=$1
     if docker ps | grep ":${PORT}" | grep -q 'redis'; then
-        open redis://127.0.0.1:${PORT}
+        ${command} redis://127.0.0.1:${PORT}
     elif docker ps | grep ":${PORT}" | grep -q 'postgres'; then
-        open postgresql://postgres:changethis@127.0.0.1:${PORT}/app
+        ${command} postgresql://postgres:changethis@127.0.0.1:${PORT}/app
     else
-        open http://localhost:${PORT}
+        ${command} http://localhost:${PORT}
```
Comment (rasswanth-s) [inline scripts/dev_tools.sh:20]: Does TablePlus also open in Linux with xdg-open command?

Comment (rAlexandre00) [reply scripts/dev_tools.sh:20]: It does. But you have to associate the url with TablePlus using xdg-mime.

Comment (rAlexandre00) [reply scripts/dev_tools.sh:20]: Example: `xdg-mime default tableplus.desktop x-scheme-handler/postgresql`

Comment (rasswanth-s) [reply scripts/dev_tools.sh:20]: I see Wonderful

Comment (madhavajay) [reply scripts/dev_tools.sh:20]: @rAlexandre00 awesome PR thank you so much. Can we add this scheme association command in a comment at the top of the script for others to use?

**Pull request merged**